addons/mail/static/src/utils/common/format.js

```diff
@@ -45,6 +47,9 @@ export async function prettifyMessageContent(rawBody, validRecords = []) {
     body = generateMentionsLinks(body, validRecords);
     body = parseAndTransform(body, addLink);
     body = await _generateEmojisOnHtml(body);
+    if (isEdited) {
+        body = generateEditLabel(body);
+    }
```

Comment:
No, the technique is not to insert `(edited)` in plain inside message body:
Notably, this doesn't work for various translation (it uses "edited" of sender/editor, so if sender is french speaker everyone will see `"édité"`).

What we should do instead is store an invisible element inside body, e.g. `data-oe-type="isMessageEdited"` which, when present and truthy, enrich the message in UI (see `Message.prepareMessageBody()`) to add "(edited)" label at end of message.

Reply:
Note that I don't think we specifically need to "prepare body" to enrich the UI.

The best would be to detect the tag/attribute in a compute field in the model, and adapt the template accordingly.

We could even imagine adding data attribute for edit date and edit partner/guest.

Reply:
As long as there's minimal data hiddenly saved inside message body, then whether it's with `prepareMessageBody` or computed JS fields is more a matter of subjective taste. I think the simplest with less LOCs is with `prepareMessageBody`, the compute approach is more LOCs and harder to grasp the immediate value of this approach.

Reply:
We don't want to dynamically generate HTML, this should be kept to minimum. It's prone to security and performance issues.

Better have a field and render HTML with a template.